README.md

```diff
@@ -0,0 +1,126 @@
+
+   ```properties
+   spring.datasource.url=jdbc:mysql://localhost:3306/online_book_store
+   spring.datasource.username=your_username
+   spring.datasource.password=your_password
+   
+## Environment Variables
+Create a .env file in the root directory of your project with the following content:
+
+MYSQLDB_DATABASE=your_database_name  
+MYSQLDB_USER=your_user_name  
+MYSQLDB_ROOT_PASSWORD=your_root_password  
+MYSQLDB_LOCAL_PORT=your_local_port    
+MYSQLDB_DOCKER_PORT=your_docker_port  
+
+SPRING_LOCAL_PORT=your_local_port  
+SPRING_DOCKER_PORT=your_docker_port  
+DEBUG_PORT=your_debug_port  
+
+JWT_SECRET=your_jwt_secret  
+JWT_EXPIRATION=your_expiration_time  
+
+## Building and Running the Project ⚙️
+
+1. - **Clone the repository**:
+  ```bash
+git clone https://github.com/slizko1/Online-Book-Store.git
+cd Online-Book-Store
+  ```
+
+2. - **Build the project**:
+  ```bash
+   mvn clean package
+  ```
+
+3. **Build and start Docker containers**:
+   ```bash
+    docker-compose down --rmi all
+   ```
+   You can use this command for remove all unnecessary images
+   ```bash
+   docker-compose up --build   
+    ```
+   This command for build new images and run app in docker
+
+### Accessing the Application
+
+- Web Application: Open your browser and go to http://localhost:8080 to access the web application.
+- Swagger UI: The API documentation is available at http://localhost:8080/swagger-ui.html.
+
+### Authentication
+
+The application uses JWT for secured access. Use the following credentials to log in as an admin:
+
+    Login: admin@mail.com
+    Password: 123456
+
+## 📬 Postman Collection 📨
+To facilitate API testing, a Postman collection is provided. You can import this collection into Postman to test the various endpoints.
+
+Download Postman Collection: [ Postman Collection Link](https://www.postman.com/joint-operations-cosmologist-51875172/workspace/online-book-store)
+
+## 🛠️ Challenges and Solutions
+
+### Challenges:
+Data Integrity: Ensuring data consistency across different parts of the application.  
+Security: Implementing robust security measures to protect user data and transactions.  
+Scalability: Designing the system to handle a growing number of users and transactions.  
+
+### Solutions:
+Spring Data JPA: Used for managing database interactions and ensuring data integrity.  
+Spring Security: Integrated to manage authentication and authorization.  
+Modular Design: Adopted a modular approach to easily scale different parts of the application independently.  
+
+## Conclusion 🏁
+The Online Book Store provides a comprehensive solution for users to manage their book shopping experience and for administrators to manage the store's inventory and orders. The project leverages robust technologies and tools to ensure a secure and scalable platform. I welcome your feedback and contributions to further enhance this application.
+
+Feel free to reach out if you have any questions or need further assistance. Happy coding! 😊
```

Comment:
Reach out who? Is there link how to connect with you? Please attach linkedin and email